hexapod_description/urdf/modules/hexaforce.xacro

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <!-- XACRO PROPERTIES -->
+  <xacro:property name="PI" value="3.14159265359" />
+  <xacro:property name="PI_2" value="1.57079632679" />
+  <xacro:property name="PI_4" value="0.78539816339" />
+  <xacro:property name="scale" value="0.001" />
+  <xacro:property name="body_mass" value="1.031"/>
+  <xacro:property name="body_length" value="0.24"/>
+  <xacro:property name="body_width" value="0.119"/>
+  <xacro:property name="body_width_extra" value="0.199"/>
+  <xacro:property name="body_height" value="0.04"/>
+
+  <xacro:property name="leg1_width" value="0.044"/>
+  <xacro:property name="leg1_length" value="0.06"/>
+  <xacro:property name="leg1_dist" value="0.2"/>
+  <xacro:property name="leg1_mass" value="0.02"/>
+
+  <xacro:property name="leg2_width" value="0.038"/>
+  <xacro:property name="leg2_length" value="0.0835"/>
+  <xacro:property name="leg2_dist" value="0.2"/>
+  <xacro:property name="leg2_mass" value="0.184"/>
+
+  <xacro:property name="leg3_width" value="0.054"/>
+  <xacro:property name="leg3_length" value="0.065"/>
+  <xacro:property name="leg3_height" value="0.10"/>
+  <xacro:property name="leg3_tip_radius" value="0.02"/>
+  <xacro:property name="leg3_mass" value="0.04"/>
+
+  <xacro:property name="force_y_offset" value="${(11.25+27)*0.001}"/>
+  <xacro:property name="force_z_offset" value="0.115"/>
+
+  <xacro:property name="joint_lim" value="${PI_2}"/>
+  <xacro:property name="joint_eff" value="30.0"/>
+  <xacro:property name="joint_vel" value="7.0"/>
```

Comment:
We should also look for the proper velocities..

Reply:
The no load speed is 55rpm (at 12V). I guess the unit is radians per seconds, so it would be 55*2pi /60 rad/s. Sound right ?

Reply:
> The no load speed is 55rpm (at 12V). I guess the unit is radians per seconds, so it would be 55*2pi /60 rad/s. Sound right ?

Exactly!